CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+Version 3.0.0 - February 19, 2020
+================================
+- Refactored airship class name to airship_flutter to resolve publish warning
```

Comment:
Do you mean "file name" here? Seems like the class is still Airship.

Reply:
Did we still need to mention what customers need to do in order to update? I assume we at least want to say something about changing import statements.

Reply:
it's the dart interface, I guess it's not technically a class